src/solver/conv_winoRxS.cpp

```diff
@@ -190,31 +189,41 @@ inline bool IsShaderContraintsMet(const ExecutionContext& ctx,
         if(!(0 <= problem.GetBackwardPadH() && problem.GetBackwardPadH() < std::pow(2, 16)))
             return false;
     }
-    const auto grid_workgroup_count_x = ctx.GetStream().GetMaxHardwareComputeUnits();
     if(!problem.IsLayoutDefault())
     {
         return false;
     }
 
+    uint64_t o_K_stride      = OH * OW;
+    uint64_t o_N_stride      = o_K_stride * K;
+    uint64_t o_N_stride_OHOW = o_N_stride + o_K_stride;
+
+    uint64_t d_C_stride    = H * W;
```

Comment:
Same as above. need `static_cast<uint64_t>()` or we have `bugprone-implicit-widening-of-multiplication-result`